pkg/versiongetter/github_release.go

```diff
@@ -63,14 +63,19 @@ func (g *GitHubReleaseVersionGetter) Get(ctx context.Context, pkg *registry.Pack
 	}
 }
 
-func (g *GitHubReleaseVersionGetter) List(ctx context.Context, pkg *registry.PackageInfo, filters []*Filter) ([]*fuzzyfinder.Item, error) {
+func (g *GitHubReleaseVersionGetter) List(ctx context.Context, pkg *registry.PackageInfo, filters []*Filter, limit int) ([]*fuzzyfinder.Item, error) {
 	repoOwner := pkg.RepoOwner
 	repoName := pkg.RepoName
 	opt := &github.ListOptions{
-		PerPage: 30, //nolint:gomnd
+		PerPage: ghMaxPerPage, //nolint:gomnd
+	}
+	if limit > 0 && opt.PerPage > limit {
```

Comment:
```suggestion
	if limit > 0 && opt.PerPage > limit && len(filters) > 0 {
```

If some filters are used, probably some releases are excluded and aqua needs to call additional API.
So I think PerPage should be ghMaxPerPage.
What do you think?

Reply:
It's a good idea, I'll make modification later.